vscode/src/extension.ts

```diff
@@ -14,13 +22,16 @@ import {
     ServerOptions
 } from 'vscode-languageclient';
 import {XMLHttpRequest} from 'xmlhttprequest-ts';
+import * as crypto from "crypto";
 
 // Track currently webview panel
 let panel: vscode.WebviewPanel | undefined = undefined;
 
 export function activate(context: ExtensionContext) {
-    let script = 'java';
-    let args = ['-jar', context.asAbsolutePath('gobpie-0.0.3-SNAPSHOT.jar')];
+    const adbSocketPath = `gobpie_adb_${crypto.randomBytes(6).toString('base64url')}.sock`
```

Comment:
I think at minimum this socket file name should start with a dot, to somewhat hide it.
Maybe it could even be created using standard temporary file and directory mechanisms, so it never clutters the project workspace.